lms/static/js/spec/verify_student/make_payment_step_view_ab_testing_spec.js

```diff
@@ -0,0 +1,225 @@
+        'use strict';
+
+        var checkPaymentButtons,
+            expectPaymentSubmitted,
+            goToPayment,
+            expectPaymentDisabledBecauseInactive,
+            expectPaymentButtonEnabled,
+            expectPriceSelected,
+            createView,
+            SERVER_ERROR_MSG = 'An error occurred!';
+
+        describe( 'edx.verify_student.MakePaymentStepView', function() {
+
+            var STEP_DATA = {
+                minPrice: '12',
+                currency: 'usd',
+                processors: ['test-payment-processor'],
+                courseKey: 'edx/test/test',
+                courseModeSlug: 'verified',
+                isABTesting: true
+            };
+
+            createView = function( stepDataOverrides ) {
+                var view = new MakePaymentStepView({
+                    el: $( '#current-step-container' ),
+                    stepData: _.extend( _.clone( STEP_DATA ), stepDataOverrides ),
+                    errorModel: new ( Backbone.Model.extend({}) )()
+                }).render();
+
+                // Stub the payment form submission
+                spyOn( view, 'submitForm' ).andCallFake( function() {} );
+                return view;
+            };
+
+            expectPriceSelected = function( price ) {
+                var sel = $( 'input[name="contribution"]' );
+
+                // check that contribution value is same as price given
+                expect( sel.length ).toEqual(1);
+                expect( sel.val() ).toEqual(price);
+            };
+
+            expectPaymentButtonEnabled = function( isEnabled ) {
+                var el = $( '.payment-button'),
+                    appearsDisabled = el.hasClass( 'is-disabled' ),
+                    isDisabled = el.prop( 'disabled' );
+
+                expect( appearsDisabled ).not.toEqual( isEnabled );
+                expect( isDisabled ).not.toEqual( isEnabled );
+            };
+
+            expectPaymentDisabledBecauseInactive = function() {
+                var payButton = $( '.payment-button' );
+
+                // Payment button should be hidden
+                expect( payButton.length ).toEqual(0);
+            };
+
+
+            goToPayment = function( requests, kwargs ) {
+                var params = {
+                    contribution: kwargs.amount || '',
+                    course_id: kwargs.courseId || '',
+                    processor: kwargs.processor || '',
+                    sku: kwargs.sku || ''
+                };
+
+                // Click the "go to payment" button
+                $( '.payment-button' ).click();
+
+                // Verify that the request was made to the server
+                AjaxHelpers.expectPostRequest(
+                    requests, '/verify_student/create_order/', $.param( params )
+                );
+
+                // Simulate the server response
+                if ( kwargs.succeeds ) {
+                    // TODO put fixture responses in the right place
```

Comment:
@rlucioni This file is the copy of this file. https://github.com/edx/edx-platform/blob/master/lms/static/js/spec/verify_student/make_payment_step_view_spec.js#L83

That's why comment is still here.